types/node/http.d.ts

```diff
@@ -207,7 +207,7 @@ declare module "http" {
             | undefined;
         defaultPort?: number | string | undefined;
         family?: number | undefined;
-        headers?: OutgoingHttpHeaders | undefined;
+        headers?: OutgoingHttpHeaders | ReadonlyArray<string> | undefined;
```

Comment:
Nit: Not enforced, but the convention for the package is along the lines of [`array-simple`](https://typescript-eslint.io/rules/array-type/#array-simple) – ie. `readonly X[]` rather than `ReadonlyArray<X>` for simple array types.

Reply:
Fixed